src/codegen/riscv64/macro-assembler-riscv64.h

```diff
@@ -378,31 +378,31 @@ class V8_EXPORT_PRIVATE TurboAssembler : public TurboAssemblerBase {
   void instr(Register rs, Register rt) { instr(rs, Operand(rt)); } \
   void instr(Register rs, int32_t j) { instr(rs, Operand(j)); }
 
-  DEFINE_INSTRUCTION(Addu)
-  DEFINE_INSTRUCTION(Daddu)
-  DEFINE_INSTRUCTION(Div)
-  DEFINE_INSTRUCTION(Divu)
-  DEFINE_INSTRUCTION(Ddivu)
-  DEFINE_INSTRUCTION(Mod)
-  DEFINE_INSTRUCTION(Modu)
-  DEFINE_INSTRUCTION(Ddiv)
-  DEFINE_INSTRUCTION(Subu)
-  DEFINE_INSTRUCTION(Dsubu)
-  DEFINE_INSTRUCTION(Dmod)
-  DEFINE_INSTRUCTION(Dmodu)
-  DEFINE_INSTRUCTION(Mul)
-  DEFINE_INSTRUCTION(Mulh)
-  DEFINE_INSTRUCTION(Mulhu)
-  DEFINE_INSTRUCTION(Dmul)
-  DEFINE_INSTRUCTION(Dmulh)
-  DEFINE_INSTRUCTION2(Mult)
-  DEFINE_INSTRUCTION2(Dmult)
-  DEFINE_INSTRUCTION2(Multu)
-  DEFINE_INSTRUCTION2(Dmultu)
-  DEFINE_INSTRUCTION2(Div)
-  DEFINE_INSTRUCTION2(Ddiv)
-  DEFINE_INSTRUCTION2(Divu)
-  DEFINE_INSTRUCTION2(Ddivu)
+  DEFINE_INSTRUCTION(Add32)
+  DEFINE_INSTRUCTION(Add64)
+  DEFINE_INSTRUCTION(Div32)
+  DEFINE_INSTRUCTION(Divu32)
+  DEFINE_INSTRUCTION(Divu64)
+  DEFINE_INSTRUCTION(Mod32)
+  DEFINE_INSTRUCTION(Modu32)
+  DEFINE_INSTRUCTION(Div64)
+  DEFINE_INSTRUCTION(Sub32)
+  DEFINE_INSTRUCTION(Sub64)
+  DEFINE_INSTRUCTION(Mod64)
+  DEFINE_INSTRUCTION(Modu64)
+  DEFINE_INSTRUCTION(Mul32)
+  DEFINE_INSTRUCTION(Mulh32)
+  DEFINE_INSTRUCTION(Mulhu32)
+  DEFINE_INSTRUCTION(Mul64)
+  DEFINE_INSTRUCTION(Mulh64)
+  // DEFINE_INSTRUCTION2(Mult)
+  // DEFINE_INSTRUCTION2(Dmult)
+  // DEFINE_INSTRUCTION2(Multu)
+  // DEFINE_INSTRUCTION2(Dmultu)
```

Comment:
I think these should be deleted. They don't map to anything useful on RISC-V.

Reply:
Removed